cartridge/vshard-utils.lua

```diff
@@ -520,7 +518,7 @@ local function get_vshard_config(group_name, conf)
         rebalancer_max_sending = vshard_groups[group_name].rebalancer_max_sending,
         sched_ref_quota = vshard_groups[group_name].sched_ref_quota,
         sched_move_quota = vshard_groups[group_name].sched_move_quota,
-        collect_lua_garbage = vshard_groups[group_name].collect_lua_garbage,
+        collect_lua_garbage = vshard_groups[group_name].collect_lua_garbage or false,
```

Comment:
If user specifies `nil` here (or even omits at all) value shouldn't be converted to false